charts/wildfly/README.md

```diff
@@ -178,6 +178,12 @@ If the Helm chart is only used to build the application image, you can skip the
 | `deploy.envFrom` | Freeform `envFrom` items | - | [Kubernetes documentation](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/).  These environment variables will be used when the application is _running_. If you need to specify environment variables when the application is built, use `build.envFrom` instead. |
 | `deploy.extraContainers` | Freeform extra `containers` items | - | [Kubernetes Documentation](https://kubernetes.io/docs/concepts/workloads/pods/#pod-templates) |
 | `deploy.imagePullSecrets` | Names of secrets to pull the application image from an private image registry | - | [Kubernetes Documentation](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod) |
+| `deploy.ingress` | Configuration specific to the creation of a `Ingress` resource to expose the application | - | - |
```

Comment:
can you link to https://kubernetes.io/docs/concepts/services-networking/ingress/ please?